AdSecGH/Components/3_Section/EditSection.cs

```diff
@@ -52,26 +55,32 @@ protected override void RegisterOutputParams(GH_OutputParamManager pManager) {
       pManager.AddGenericParameter("Profile", "Pf", "Profile defining the Section solid boundary", GH_ParamAccess.item);
       pManager.AddGenericParameter("Material", "Mat", "Material for the section", GH_ParamAccess.item);
       pManager.AddGenericParameter("DesignCode", "Code", "Section DesignCode", GH_ParamAccess.item);
-      pManager.AddGenericParameter("RebarGroup", "RbG", "Reinforcement Groups in the section (applicable for only concrete material).", GH_ParamAccess.list);
-      pManager.AddGenericParameter("SubComponent", "Sub", "Subcomponents contained within the section", GH_ParamAccess.list);
-      pManager.AddGenericParameter("SectionCurves", "CAD", "All curves used for displaying the section - useful for making CAD drawings", GH_ParamAccess.list);
+      pManager.AddGenericParameter("RebarGroup", "RbG",
+        "Reinforcement Groups in the section (applicable for only concrete material).", GH_ParamAccess.list);
+      pManager.AddGenericParameter("SubComponent", "Sub", "Subcomponents contained within the section",
+        GH_ParamAccess.list);
+      pManager.AddGenericParameter("SectionCurves", "CAD",
+        "All curves used for displaying the section - useful for making CAD drawings", GH_ParamAccess.list);
       pManager.HideParameter(7);
     }
 
     protected override void SolveInstance(IGH_DataAccess DA) {
       // 0 section
-      AdSecSection in_section = AdSecInput.AdSecSection(this, DA, 0);
+      var in_section = AdSecInput.AdSecSection(this, DA, 0);
       if (in_section == null) {
-        AddRuntimeMessage(GH_RuntimeMessageLevel.Warning, "Input parameter " + Params.Input[0].NickName + " failed to collect data!");
+        AddRuntimeMessage(GH_RuntimeMessageLevel.Warning,
+          "Input parameter " + Params.Input[0].NickName + " failed to collect data!");
         return;
       }
+
       // 1 profile
       AdSecProfileGoo profile = null;
       if (Params.Input[1].SourceCount > 0) {
         profile = AdSecInput.AdSecProfileGoo(this, DA, 1, true);
       } else {
         profile = new AdSecProfileGoo(in_section.Section.Profile, in_section.LocalPlane);
       }
+
       DA.SetData(1, profile);
```

Comment:
why not to set all data at the end of the method?

Reply:
this file are from autoformat, out of scope for this PR